.github/release-drafter.yml

```diff
@@ -0,0 +1,60 @@
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+
+template: |
+  # What's Changed
+
+  $CHANGES
+
+  **Full Changelog**: https://github.com/$OWNER/$REPOSITORY/compare/$PREVIOUS_TAG...v$RESOLVED_VERSION
+
+categories:
+  - title: 'Breaking'
+    label: 'type: breaking'
+
+  - title: 'New'
+    label: 'type: feature'
+
+  - title: 'Bug Fixes'
+    label: 'type: bugfix'
+
+  - title: 'Maintenance'
+    labels:
+      - 'type: maintenance'
+      - 'type: performance'
+
+  - title: 'Documentation'
+    label: 'type: documentation'
+
+  - title: 'Other changes'
```

Comment:
I would prefer to remove this. Other changes would cause a problem in the future. I guess we addressed all categories.